src/_gettsim/transfers/erwerbsm_rente.py

```diff
@@ -0,0 +1,358 @@
+    pensioners are credited with additional earning points.
+    They receive their average earned income points for
+    each year between their age of retirement and the "zurechnungszeitsgrenze".
+
+    Parameters
+    ----------
+    entgeltp_west
+        See basic input variable :ref:`entgeltp_west <entgeltp_west>
+    entgeltp_ost
+        See basic input variable :ref:`entgeltp_ost <entgeltp_ost>
+    entgeltp_zurechnungszeit
+        See :func:`entgeltp_zurechnungszeit`.
+
+    Returns
+    -------
+    Final pension points for Erwerbsminderungsrente (public disability insurance)
+
+    """
+
+    anteil_entgeltp_ost = entgeltp_ost / (entgeltp_west + entgeltp_ost)
+
+    out = entgeltp_ost + (entgeltp_zurechnungszeit * anteil_entgeltp_ost)
+
+    return out
+
+
+def entgeltp_zurechnungszeit(
+    durchschn_entgeltp: float,
+    age_of_retirement: float,
+    erwerbsm_rente_params: dict,
+) -> float:
+    """Additional Entgeltpunkte accumulated through "Zurechnungszeit" for
+    Erwerbsminderungsrente (public disability insurance)
+    In the case of the public disability insurance,
+    pensioners are credited with additional earning points.
+    They receive their average earned income points for
+    each year between their age of retirement and the "zurechnungszeitsgrenze".
+
+    Parameters
+    ----------
+    durchschn_entgeltp
+        See :func:`durchschn_entgeltp`.
+    age_of_retirement
+        See :func:`age_of_retirement`.
+    erwerbsm_rente_params
+        See params documentation :ref:`erwerbsm_rente_params <erwerbsm_rente_params>.
+
+
+    Returns
+    -------
+    Final pension points for Erwerbsminderungsrente (public disability insurance)
+
+    """
+    zurechnungszeitsgrenze = erwerbsm_rente_params["zurechnungszeitsgrenze"]
+
+    out = (zurechnungszeitsgrenze - (age_of_retirement)) * durchschn_entgeltp
+
+    return out
+
+
+def durchschn_entgeltp(
+    entgeltp_west: float,
+    entgeltp_ost: float,
+    age_of_retirement: float,
+    erwerbsm_rente_params: dict,
+) -> float:
+    """Average earning points as part of the "Grundbewertung".
+    Earnings points are divided by "belegungsfähige Gesamtzeitraum" which is
+    the period from the age of 17 until the start of the pension.
+
+    Legal reference: SGB VI § 72: Grundbewertung
+
+    Parameters
+    ----------
+    entgeltp_west
+        See basic input variable :ref:`entgeltp_west <entgeltp_west>
+    entgeltp_ost
+        See basic input variable :ref:`entgeltp_ost <entgeltp_ost>
+    age_of_retirement
+        See :func:`age_of_retirement`.
+    erwerbsm_rente_params
+        See params documentation :ref:`erwerbsm_rente_params <erwerbsm_rente_params>.
+
+    Returns
+    -------
+    average entgeltp
+    """
+
+    beleg_gesamtzeitr = (
+        age_of_retirement - erwerbsm_rente_params["altersgrenze_grundbew"]
+    )
+
+    entgeltp_gesamt = entgeltp_west + entgeltp_ost
+
+    durchschn_entgeltp = entgeltp_gesamt / beleg_gesamtzeitr
```

Comment:
```suggestion
    durchschn_entgeltp = (entgeltp_west + entgeltp_ost) / beleg_gesamtzeitr
```

Easier to see what is happening this way.